src/Analytics.js

```diff
@@ -147,6 +148,23 @@ class Analytics {
         return true;
     }
 
+    startPageChangeTimer() {
+        performance.clearMarks('riot_page_change_start');
```

Comment:
You'll have to use these APIs conditionally: they aren't available in all browsers.

Reply:
oh yes - MDN lied to me! Fair enough then.